pkg/fuse.go

```diff
@@ -208,16 +212,7 @@ func (f *billyFuse) GetInodeAttributes(ctx context.Context, op *fuseops.GetInode
 		return fuse.ENOENT
 	}
 	op.Attributes = infoToAttributes(inode.info)
-	op.AttributesExpiration = time.Now().Add(time.Second)
-	return nil
-}
-
-func (f *billyFuse) OpenDir(ctx context.Context, op *fuseops.OpenDirOp) error {
-	log.Println("fuse OpenDir()")
-	return nil
-}
-
-func (f *billyFuse) ReleaseDirHandle(ctx context.Context, op *fuseops.ReleaseDirHandleOp) error {
+	op.AttributesExpiration = latest
```

Comment:
If your underlying git repository changes, you can detect that and invalidate the kernel caches manually.

Reply:
Nice! Good idea.